modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1048,7 +1048,7 @@ class Edit_User extends \NDB_Form
                 $password = new \Password($values['Password_hash']);
                 // New password must be different than current one
                 $passwordChanged = \User::factory($this->identifier)
-                    ->passwordChanged($_POST['password']);
+                    ->passwordChanged($password);
```

Comment:
```suggestion
                    ->passwordChanged($_POST['Password_hash']);
```

I believe this should fix your problem @kongtiaowang

Reply:
@johnsaigle This is not correct. In this function only take care values 
function _process($values).

Reply:
This fixed an issue for me when I tried to change a user's password on `major`.

It would be helpful if you described what error you're getting.

Reply:
[Wed Oct 23 14:18:34.013500 2019] [php7:error] [pid 7294] [client 192.168.122.1:53884] PHP Fatal error:  Uncaught TypeError: Argument 1 passed to User::passwordChanged() must be of the type string, null given, called in /var/www/Loris/modules/user_accounts/php/edit_user.class.inc on line 1051 and defined in /var/www/Loris/php/libraries/User.class.inc:483\nStack trace:\n#0 /var/www/Loris/modules/user_accounts/php/edit_user.class.inc(1051): User->passwordChanged(NULL)\n#1 /var/www/Loris/php/libraries/LorisForm.class.inc(1324): LORIS\\user_accounts\\Edit_User->_validateEditUser(Array)\n#2 /var/www/Loris/php/libraries/NDB_Form.class.inc(132): LorisForm->validate()\n#3 /var/www/Loris/php/libraries/NDB_Form.class.inc(192): NDB_Form->save()\n#4 /var/www/Loris/modules/user_accounts/php/edit_user.class.inc(579): NDB_Form->handle(Object(Zend\\Diactoros\\ServerRequest))\n#5 /var/www/Loris/src/Middleware/UserPageDecorationMiddleware.php(187): LORIS\\user_accounts\\Edit_User->handle(Object(Zend\\Diactoros\\ServerRequest))\n#6 /var/www/Loris/src/Middleware/PageDecorationMiddleware.php(53): LORIS\\Middleware\\UserPageDecorationMiddleware->proce in /var/www/Loris/php/libraries/User.class.inc on line 483, referer: https://test-loris-dev.loris.ca/user_accounts/edit_user/

Reply:
Yeah that's what I got when I tried to test. `$_POST['Password_hash']` or `$values['Password_hash']` should fix that.